### ISSUE TITLE
UnusedLocalVariable doesn't detect unused var when defined twice

### ISSUE THREAD

**Issue by bkdotcom** [phpmd/phpmd#540]
```PHP
function testUnused()
{
    $worthlessVar = false;
    $worthlessVar = true;
}
```

phpmd fails to raise any UnusedLocalVariable notice re `$worthlessVar`

**Comment by kylekatarnls** [comment]
The following:
```php
function testUnused()
{
    $worthlessVar = false;
}
```
did trigger the UnusedLocalVariable so it's actually the second assignation which is seen as a usage.

**Comment by bkdotcom** [comment]
Excellent!
